Don't run iscsi-iname to generate a new InitiatorName if the destination does
not contain a /etc/iscsi/initiatorname.iscsi file for the following reasons:
* In Chromium OS, the InitiatorName file is located at
  /var/lib/iscsi/initiatorname.iscsi
* When cros-compiling, the iscsi-iname executable may be incompatible with the
  host processor
* The file will be created based on the Vendor Encapsulated Options from DHCP

Fix by Victor Dodon <printesoi@chromium.org>
--- a/Makefile
+++ b/Makefile
@@ -69,13 +69,13 @@ clean:
 # note that make may still execute the blocks in parallel
 .NOTPARALLEL: install_user install_programs install_initd \
 	install_initd_suse install_initd_redhat install_initd_debian \
-	install_etc install_iface install_doc install_kernel install_iname
+	install_etc install_iface install_doc install_kernel
 
 install: install_programs install_doc install_etc \
-	install_initd install_iname install_iface
+	install_initd install_iface
 
 install_user: install_programs install_doc install_etc \
-	install_initd install_iname install_iface
+	install_initd install_iface
 
 install_programs:  $(PROGRAMS)
 	$(INSTALL) -d $(DESTDIR)$(sbindir)
@@ -127,15 +127,6 @@ install_doc: $(MANPAGES)
 install_kernel:
 	$(MAKE) -C kernel install_kernel
 
-install_iname:
-	if [ ! -f $(DESTDIR)$(etcdir)/iscsi/initiatorname.iscsi ]; then \
-		echo "InitiatorName=`$(DESTDIR)$(sbindir)/iscsi-iname`" > $(DESTDIR)$(etcdir)/iscsi/initiatorname.iscsi ; \
-		echo "***************************************************" ; \
-		echo "Setting InitiatorName to `cat $(DESTDIR)$(etcdir)/iscsi/initiatorname.iscsi`" ; \
-		echo "To override edit $(etcdir)/iscsi/initiatorname.iscsi" ; \
-		echo "***************************************************" ; \
-	fi
-
 depend:
 	for dir in usr utils utils/fwparam_ibft; do \
 		$(MAKE) -C $$dir $@; \
